### PR TITLE
Add Upsolver user testimonial

### PR DESCRIPTION
**Pull request by mosabua**

### CODE REVIEW

File: _data/users.yml [hunk 1]
```diff
@@ -964,6 +964,23 @@
...
+  anchor: upsolver
+  logo: /assets/images/logos/upsolver.png
+  logosmall: /assets/images/logos/upsolver-small.png
+  testimonial: |
+    Upsolver is the self-serve cloud data ingestion service for high-scale
+    workloads such as big data, streaming, and AI. Our unique approach offers
+    you no-code and low-code options to easily create pipelines for high-scale
+    data movement in minutes.
+
+    Upsolver supports Trino and is a regular sponsor of Trino events.
```
Comment (mosabua) [inline _data/users.yml:978]: Not sure we want to add that .. probably not

**Comment by rhasson** [comment]
Please add the following text:

[Upsolver](https://upsolver.com) is a control center for the Iceberg lakehouse. With Upsolver, users can ingest files, streams and CDC data from popular sources into Apache Iceberg tables. Users can also build SQL ETL pipelines to transform, enrich and model data to fit business needs. Upsolver continuously optimizes and maintains Iceberg tables, created by Upsolver and other engines, to give Trino users best query experience. 

Upsolver integrates with AWS Glue Data Catalog, Apache Hive, Snowflake Polaris and compatible Iceberg REST catalogs enabling concurrent reads and write, ACID transactions and continuous optimizations. Upsolver integrates with Trino's Merge-On-Read support for Iceberg utilizing equality deletes for low latency data freshness and highly performant queries on frequently changing data, like CDC streams from operational databases. 

Customers like Ironsource - a Unity company, AppsFlyer and Cox Automotive use Upsolver to ingest, transform and manage their PB-size lakehouse, querying it with Amazon Athena managed Trino.

**Comment by colebow** [comment]
> Please add the following text:
> 
> [Upsolver](https://upsolver.com) is a control center for the Iceberg lakehouse. With Upsolver, users can ingest files, streams and CDC data from popular sources into Apache Iceberg tables. Users can also build SQL ETL pipelines to transform, enrich and model data to fit business needs. Upsolver continuously optimizes and maintains Iceberg tables, created by Upsolver and other engines, to give Trino users best query experience.
> 
> Upsolver integrates with AWS Glue Data Catalog, Apache Hive, Snowflake Polaris and compatible Iceberg REST catalogs enabling concurrent reads and write, ACID transactions and continuous optimizations. Upsolver integrates with Trino's Merge-On-Read support for Iceberg utilizing equality deletes for low latency data freshness and highly performant queries on frequently changing data, like CDC streams from operational databases.
> 
> Customers like Ironsource - a Unity company, AppsFlyer and Cox Automotive use Upsolver to ingest, transform and manage their PB-size lakehouse, querying it with Amazon Athena managed Trino.

I think this likely is too long and too marketing-heavy for a user testimonial on an open source website. The objective here is not sell a product, but to inform Trino users how you use and integrate with Trino. If you check the user page - https://trino.io/users - most user testimonials keep it to ~50-75 words, and the two lengthy ones are the primary vendors offering Trino directly.

**Comment by rhasson** [comment]
Thanks for guidance @colebow see revised text:

Upsolver_logo_2024
### Upsolver
Upsolver is a Lakehouse management solution used to ingest, transform and optimize Apache Iceberg and Hive tables on object stores. Upsolver is closely integrated with Trino to provide the most optimal performance for tables of any size over batch and streaming data.

- [Visit the Upsolver website](https://upsolver.com)
- [Get started with Apache Iceberg](https://docs.upsolver.com/content/get-started/apache-iceberg)

**Review by colebow — APPROVED** [review]

**Pull request merged**